packages/react/src/hooks/useWindowSize.js

```diff
@@ -0,0 +1,44 @@
+import { useState, useEffect, useRef } from 'react';
+
+const DEBOUNCE_TIMEOUT = 300;
+
+const useWindowSize = (debounceDelay = DEBOUNCE_TIMEOUT) => {
+  const [windowSize, setWindowSize] = useState({
+    width: window?.innerWidth,
+    height: window?.innerHeight,
+  });
+  const timer = useRef(null);
+
+  const debounce = (fn, delay) => {
```

Comment:
lodash is available within the project, so you can easily replace this with `import debounce from 'lodash/debounce';`

Reply:
Done 👍